src/mpl/src/ParquetFP/src/DB.cxx

```diff
@@ -1800,6 +1800,7 @@ float DB::evalSteiner(bool useWts, bool scaleTerms)
 
   float total = 0.f;
   vector<Point> pointsOnNet;
+  Flute f = Flute();
```

Comment:
I'm not a big fan of single char names for variables. I think you can just use `Flute flute;` for clarity.